math/clingo/Portfile

```diff
@@ -3,10 +3,12 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           muniversal 1.0
```

Comment:
are we sure that this is needed - I thought that building using the `cmake` PG takes care of the universal stuff

Reply:
@reneeotten No, `cmake` PG does not do that (or if it should, then it is broken).

Reply:
well, I'm pretty sure it works for many other ports that do include the `cmake` PG but not the `muniversal` one. What's the issue you're seeing - if you post it here, perhaps someone can shed some light on it.

Reply:
> well, I'm pretty sure it works for many other ports that do include the `cmake` PG but not the `muniversal` one. What's the issue you're seeing - if you post it here, perhaps someone can shed some light on it.

If `muniversal` is removed, the build simply ignores `+universal` and builds a single arch.

Reply:
We've mentioned in several places before that we don't really want to add the muniversal portgroup to every single port just to support building universal on PowerPC systems to accommodate gcc's lack of support for multiple arch flags. The muniversal portgroup can cause additional issues in some ports. It would be a better use of time to fix the gcc ports so that gcc can accept multiple arch flags, by incorporating a fixed or rewritten equivalent of Apple's driver driver.